Source/Extensions/Blazorise.Components/Autocomplete.razor

```diff
@@ -62,21 +62,34 @@
             for ( int i = 0; i < FilteredData.Count; ++i )
             {
                 var item = FilteredData[i];
+                var isActiveItem = IsActiveItem(item, i);
 
                 var text = GetItemText( item );
                 var value = GetItemValue( item );
 
                 @if ( ItemContent is not null )
                 {
-                    <DropdownItem ElementId="@DropdownItemId(i)" @key="item" Active="@(IsActiveItem(item, i))" Class="@DropdownItemClassNames(i)" Value="@value" Clicked="@OnDropdownItemClicked">@ItemContent(new (item, value, text))</DropdownItem>
+                    <DropdownItem ElementId="@DropdownItemId(i)" @key="item" Active="@isActiveItem" Class="@DropdownItemClassNames(i)" Value="@value" Clicked="@OnDropdownItemClicked">
+                        @if ( SuggestMultipleCheckbox )
+                        {
+                            <Field Horizontal>
+                                <Check @onclick:stopPropagation TValue="bool" Checked="@isActiveItem"></Check>
+                                @ItemContent(new (item, value, text))
```

Comment:
Should `@ItemContent(new (item, value, text))` be inside of Check?

Reply:
I don't think so... It might be harder to handle complex markup styling inside the checkbox then outside? What do you think?

But maybe for the regular @text, it makes sense to go inside instead.

Reply:
Check is made to hold the content and to place it where belongs, HTML structure-wise, based on the used provider.